packages/paste-core/components/visual-picker/stories/index.stories.tsx

```diff
@@ -0,0 +1,1070 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable import/no-extraneous-dependencies */
+import * as React from 'react';
+import {Anchor} from '@twilio-paste/anchor';
+import {Text} from '@twilio-paste/text';
+import {useUID} from '@twilio-paste/uid-library';
+import {Avatar, AvatarGroup} from '@twilio-paste/avatar';
+import {StatusBadge} from '@twilio-paste/status';
+import {CodeIcon} from '@twilio-paste/icons/esm/CodeIcon';
+import {MediaObject, MediaBody, MediaFigure} from '@twilio-paste/media-object';
+import {Box} from '@twilio-paste/box';
+import {CommunityIcon} from '@twilio-paste/icons/esm/CommunityIcon';
+import {AcceptIcon} from '@twilio-paste/icons/esm/AcceptIcon';
+import {CustomizationProvider} from '@twilio-paste/customization';
+import {useTheme} from '@twilio-paste/theme';
+
+import {VisualPickerRadioGroup, VisualPickerRadio, VisualPickerCheckboxGroup, VisualPickerCheckbox} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Visual Picker',
+  component: VisualPickerRadioGroup,
```

Comment:
suggestion: you might want to split these out into seperate files based on type, radio, checkbox and customization. We probably also need controlled and uncontrolled stories to prove defaultChecked and checked states render correctly for both types